_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'Bitcoin Optech Newsletter #131'
+permalink: /en/newsletters/2021/01/13/
+name: 2021-01-13-newsletter
+slug: 2021-01-13-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a new proposed Bitcoin P2P protocol
+message, a BIP for the bech32 modified address format, and an idea for
+preventing UTXO probing in proposed dual-funded LN channels.  Also
+included are our regular sections with the summary of a Bitcoin Core PR
+Review Club meeting, a list of releases and release candidates, and
+descriptions of notable changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Proposed `disabletx` message:** in 2012, [BIP37][] was published,
+  giving lightweight clients the ability to request that peers not
+  relay to the client any unconfirmed transactions until the client
+  had loaded a [bloom filter][topic transaction bloom filtering].
+  Bitcoin Core [later][bitcoin core #6993] reused this mechanism for its
+  bandwidth-saving `-blocksonly` mode where a node requests that its
+  peers don't send it any unconfirmed transactions at all.  Last year,
+  Bitcoin Core with default settings began opening a few
```

Comment:
maybe s/a few/up to two/

(it is now currently up to three, or a bit more than two, if the temporary block relay connections added in https://github.com/bitcoin/bitcoin/pull/19858 are counted)